app/src/main/res/layout/activity_registration.xml

```diff
@@ -28,7 +28,6 @@
             android:layout_width="match_parent"
             android:layout_height="@dimen/increment"
             android:background="@android:color/transparent"
-            android:elevation="4dp"
```

Comment:
This was showing up as elevated on top of the app bar, not adding elevation to the app bar itself. Removing now has the view matching the pre-androix/material update again.